### ISSUE TITLE
Please update documentation

### ISSUE THREAD

**Issue by aggaton** [suriyun-production/mmorpg-kit-docs#1133]
I was trying to follow the current documentation to add a new player model however I got stopped somewhere in the middle because apparently the components added have changed so much from when the documentation were made that it was impossible to follow. Also it seem to stop around when animations is being added and it doesn't seem to explain how to actually add it and the controller(s), just the various types. What I am looking for is to be shown how to add a skinned mech-anim model as a player character, from start to finish in the mmo demo scene.

**Comment by insthync** [comment]
You means this document ? 
https://suriyun-production.github.io/mmorpg-kit-docs/#/pages/108-animator-character-model

**Comment by aggaton** [comment]
Yes, I tried following the instructions to add a new Player Character and Model, however it seems to be incomplete and was out of sync with the current version. Slightly confusing as well if you are not familiar with the kit as I am not. It looks to be a pretty cool framework, and wanted to try and change a few things on the demo scene, however it seemed super complicated to change what seems like such a simple thing like the character model.

**Comment by insthync** [comment]
How about this https://dev.to/insthync/create-game-with-mmorpg-kit-part-2-map-and-first-character-54dl

**Comment by aggaton** [comment]
Much better, thank you!

**Comment by aggaton** [comment]
Hi, so I finally got around trying this. When I start the game for the first time, I can see the toon I added on the character selection screen, once I create a player and enter the game, no model is shown, and it keeps falling down, sliding off the cube that was created while it looses health every second. I redid this 3 times and came to the same result. Do you have any idea what might be wrong? https://drive.google.com/file/d/1JiqIPNDs45NJYaIzcBuZkewi1bnBngum/view?usp=sharing

**Comment by insthync** [comment]
May I see your colliders setup for both character and cube?

**Comment by aggaton** [comment]
Here is a video of the setup

https://drive.google.com/file/d/1d7IUjsFgup714v5UGTH5kXD2kQmX4hqA/view?usp=sharing

I'll try add a collider that covers the feet area. However it is weird the model is not even showing.

So I tried two other models, one had very small scaling and I got below error, might be due to a too small scale at 0.01556005 x,y,z. The second model I tried worked perfectly, it has a scale of 1,1,1.

Step Offset must be less or equal to <scaled Height> + <scaled Radius> * 2
UnityEngine.StackTraceUtility:ExtractStackTrace ()
MultiplayerARPG.UICharacterCreate:OnSelectCharacter (MultiplayerARPG.IPlayerCharacterData) (at Assets/UnityMultiplayerARPG/Core/Scripts/LanGame/UI/UICharacterCreate.cs:297)
MultiplayerARPG.UICharacterCreate:OnSelectCharacter (MultiplayerARPG.UICharacter) (at Assets/UnityMultiplayerARPG/Core/Scripts/LanGame/UI/UICharacterCreate.cs:282)
UnityEngine.Events.UnityEvent`1<MultiplayerARPG.UICharacter>:Invoke (MultiplayerARPG.UICharacter)
UISelectionManager`4<MultiplayerARPG.ICharacterData, MultiplayerARPG.UICharacter, MultiplayerARPG.CharacterDataEvent, MultiplayerARPG.UICharacterEvent>:Select (object) (at Assets/UnityMultiplayerARPG/Core/Scripts/UI/UISelectionManager.cs:107)
UISelectionManager`4<MultiplayerARPG.ICharacterData, MultiplayerARPG.UICharacter, MultiplayerARPG.CharacterDataEvent, MultiplayerARPG.UICharacterEvent>:Add (MultiplayerARPG.UICharacter) (at Assets/UnityMultiplayerARPG/Core/Scripts/UI/UISelectionManager.cs:52)
MultiplayerARPG.UICharacterCreate/<>c__DisplayClass65_0:<LoadCharacters>b__0 (int,MultiplayerARPG.BasePlayerCharacterEntity,UnityEngine.GameObject) (at Assets/UnityMultiplayerARPG/Core/Scripts/LanGame/UI/UICharacterCreate.cs:204)
UIList:Generate<MultiplayerARPG.BasePlayerCharacterEntity> (System.Collections.Generic.IEnumerable`1<MultiplayerARPG.BasePlayerCharacterEntity>,System.Action`3<int, MultiplayerARPG.BasePlayerCharacterEntity, UnityEngine.GameObject>) (at Assets/UnityMultiplayerARPG/Core/Scripts/UI/UIList.cs:54)
MultiplayerARPG.UICharacterCreate:LoadCharacters () (at Assets/UnityMultiplayerARPG/Core/Scripts/LanGame/UI/UICharacterCreate.cs:186)
MultiplayerARPG.UICharacterCreate:OnEnable () (at Assets/UnityMultiplayerARPG/Core/Scripts/LanGame/UI/UICharacterCreate.cs:269)
UIBase:Show () (at Assets/UnityMultiplayerARPG/Core/Scripts/UI/UIBase.cs:64)
UIHistory:Next (UIBase) (at Assets/UnityMultiplayerARPG/Core/Scripts/UI/UIHistory.cs:27)
UnityEngine.EventSystems.EventSystem:Update ()

**Comment by insthync** [comment]
I saw the `Character Controller` is disabled, you may try turning it on in prefab

![Screenshot_642](https://user-images.githubusercontent.com/2697550/133182989-1bbe9bd8-be6f-48cd-b6c0-3fc7b27d33e9.png)

**Comment by aggaton** [comment]
So I stripped out all the colliders of the first model and just went with a basic one, and the toon showed up and is not falling. I guess the mmo kit does not like the fancy collider setup I had made for each limb... However I still have an issue with the second model with the very small scale factor. Thanks for all your help btw.

![image](https://user-images.githubusercontent.com/5942273/133182944-3004aae7-9ca1-47b9-a624-640260facdd4.png)

Not sure what to change to prevent the error from being thrown. The model came like this from the asset store.
